### PR TITLE
Mandate localAccount in AccountInfo

### PR DESCRIPTION
**Pull request by sameerag**
This PR makes the `localAccountId` a mandatory property in `AccountInfo`

### CODE REVIEW

File: change/@azure-msal-common-2020-11-06-12-29-58-mandate-localAccountId.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
+  "comment": "Mandate localAccount in AccountInfo",
+  "packageName": "@azure/msal-common",
+  "email": "sameera.gajjarapu@microsoft.com",
+  "dependentChangeType": "patch",
```
Comment (tnorling) [inline change/@azure-msal-common-2020-11-06-12-29-58-mandate-localAccountId.json:6]:
```suggestion
  "dependentChangeType": "none",
```

**Review by tnorling — APPROVED** [review]

File: change/@azure-msal-common-2020-11-06-12-29-58-mandate-localAccountId.json (hunk 1 above)
Comment (sameerag) [reply change/@azure-msal-common-2020-11-06-12-29-58-mandate-localAccountId.json:6]: I swear again, I pushed this change. there is some mess up with beachball changes post script. :(

File: lib/msal-common/src/cache/entities/AccountEntity.ts [hunk 2]
```diff
@@ -69,7 +69,8 @@ export class AccountEntity {
             homeAccountId: this.homeAccountId,
             environment: this.environment,
             tenantId: this.realm,
-            username: this.username
+            username: this.username,
+            localAccountId: this.localAccountId
```
Comment (jasonnutter) [inline lib/msal-common/src/cache/entities/AccountEntity.ts:73]: Might this result in cache misses?

Comment (jasonnutter) [reply lib/msal-common/src/cache/entities/AccountEntity.ts:73]: For apps that upgrade to this version

Comment (pkanher617) [reply lib/msal-common/src/cache/entities/AccountEntity.ts:73]: localAccountId is not part of the cacheKey, it is only required as part of the interface.

**Review by pkanher617 — APPROVED** [review]

**Pull request merged**